feathr_project/feathr/client.py

```diff
@@ -217,7 +217,55 @@ def multi_get_online_features(self, feature_table, keys, feature_names):
+        For sparse array, it will be returned as tuple of index array and value array. The order of elements in the
+        arrays won't be changed.
+        """
+        typed_result = []
+        for raw_feature in feature_list:
+            if raw_feature:
+                feature_value = FeatureValue()
+                decoded = base64.b64decode(raw_feature)
+                feature_value.ParseFromString(decoded)
+                if feature_value.WhichOneof('FeatureValueOneOf') == 'booleanValue':
+                    typed_result.append(feature_value.booleanValue)
+                elif feature_value.WhichOneof('FeatureValueOneOf') == 'stringValue':
+                    typed_result.append(feature_value.stringValue)
+                elif feature_value.WhichOneof('FeatureValueOneOf') == 'floatValue':
+                    typed_result.append(feature_value.floatValue)
+                elif feature_value.WhichOneof('FeatureValueOneOf') == 'doubleValue':
+                    typed_result.append(feature_value.doubleValue)
+                elif feature_value.WhichOneof('FeatureValueOneOf') == 'intValue':
+                    typed_result.append(feature_value.intValue)
+                elif feature_value.WhichOneof('FeatureValueOneOf') == 'longValue':
+                    typed_result.append(feature_value.longValue)
+                elif feature_value.WhichOneof('FeatureValueOneOf') == 'stringArray':
+                    typed_result.append(feature_value.stringArray.strings)
+                elif feature_value.WhichOneof('FeatureValueOneOf') == 'floatArray':
+                    typed_result.append(feature_value.floatArray.floats)
+                elif feature_value.WhichOneof('FeatureValueOneOf') == 'sparseStringArray':
+                    typed_result.append((feature_value.sparseStringArray.integers, feature_value.sparseStringArray.strings))
+                elif feature_value.WhichOneof('FeatureValueOneOf') == 'sparseBoolArray':
+                    typed_result.append((feature_value.sparseBoolArray.integers, feature_value.sparseBoolArray.booleans))
+                elif feature_value.WhichOneof('FeatureValueOneOf') == 'sparseFloatArray':
+                    typed_result.append((feature_value.sparseFloatArray.integers, feature_value.sparseFloatArray.floats))
+                elif feature_value.WhichOneof('FeatureValueOneOf') == 'sparseDoubleArray':
+                    typed_result.append((feature_value.sparseDoubleArray.integers, feature_value.sparseDoubleArray.doubles))
+                elif feature_value.WhichOneof('FeatureValueOneOf') == 'sparseLongArray':
+                    typed_result.append((feature_value.sparseLongArray.index_integers, feature_value.sparseLongArray.longs))
+                else:
+                    # fail to load the type. Maybe a new type that is not supported by this client version
+                    typed_result.append(None)
+            else:
+                typed_result.append(raw_feature)
```

Comment:
This is just  typed_result.append(None), is this expected?

Reply:
yes. see the comment so it fails gracefully